clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/view/GoogleSecurityGroupProvider.groovy

```diff
@@ -180,20 +182,45 @@ class GoogleSecurityGroupProvider implements SecurityGroupProvider<GoogleSecurit
     return rangeRules.sort()
   }
 
-  static List<String> getMatchingServerGroupNames(String account,
-                                                  Set<GoogleSecurityGroup> securityGroups,
-                                                  Set<String> tags,
-                                                  String networkName) {
+  /**
+   * Calculates security group names that match account, networkName, and tags
+   * @param account - GCE account name.
+   * @param securityGroups - Set of server groups to filter.
+   * @param tags - GCE network tags to filter security groups by.
+   * @param networkName - GCE network name.
+   * @return Security group names that match account, networkName, and network tags.
+   */
+  static List<String> getMatchingSecurityGroupNames(String account,
+                                                    Set<GoogleSecurityGroup> securityGroups,
+                                                    Set<String> tags,
+                                                    String networkName) {
     tags = tags ?: [] as Set
     securityGroups?.findResults { GoogleSecurityGroup securityGroup ->
       def accountAndNetworkMatch = securityGroup.accountName == account && securityGroup.network == networkName
-      boolean targetTagsEmpty = !securityGroup.targetTags
+      if (!accountAndNetworkMatch) {
+        return null
+      }
+
+      log.info(",, security group name: ${securityGroup.name}")
```

Comment:
How often is this function called?  Seems like the `log.info` here could be `log.debug`

Reply:
These are all extraneous.